spec/features/account_reset/delete_account_spec.rb

```diff
@@ -48,8 +48,6 @@
       expect(page).to have_content t('account_reset.confirm_request.security_note')
       expect(page).to have_content t('account_reset.confirm_request.close_window')
 
-      reset_email
```

Comment:
what does this do? how did it change things?

Reply:
So Sunday Paul misread the code and thought that the email we wanted to check was sent BEFORE `reset_email`.  Monday Paul just realized that is wrong.  Not sure why the tests pass without the reset, but it is not correct.

I will take this one back to the shop.